Lunch-T-shirt-script/makePDFs.ps1

```diff
@@ -0,0 +1,179 @@
+# Input file is expected to be tab-separated with no first line for header names
+#    New line format must be Windows-style \r\n aka CRLF
+#    Column order is TEAM NAME -> PLAYER NAME -> PLAYER EMAIL -> LUNCH OPTION
+#    PLAYER NAME and LUNCH OPTION may be blank, though in the future names may not be
+# Make sure the following files are all located in the same directory as this script:
+#    BouncyCastle.Crypto.dll
+#    Common.Logging.Core.dll
+#    Common.Logging.dll
+#    itext.html2pdf.dll
+#    itext.io.dll
+#    itext.kernel.dll
+#    itext.layout.dll
+#    itext.styledxmlparser.dll
+
+# ---- CHANGE THESE VARIABLES AS NECESSARY ----
+
+$inputFile = "E:\OneDrive\Puzzle Day\lunch-script\input.txt"

```

Comment:
Does it work to make these command line parameters? Most folks probably don't have an E drive

Reply:
I could, but I put the whole file path in there from my local machine on purpose so the script will explicitly fail unless somebody actually sets the input and output files correctly.  Also, for user-proofing purposes, I want them to do that from within the file instead of from the command line so they also read and confirm that the input file conforms to the expected format.